antiope-s3/src/Antiope/S3.hs

```diff
@@ -139,10 +140,18 @@ lsBucketPage (Just req) = do
       Just True -> Just $ nextPageReq req resp
       _         -> Nothing
 
--- | Streams the entire set of results (i.e. all pages) of a ListObjectsV2
+-- | Streams all pages of the result (ListObjectsV2Responses) of a ListObjectsV2
+-- request from S3.
+-- lsBucketResponseStream :: MonadAWS m => ListObjectsV2 -> ConduitT i ListObjectsV2Response m ()
+lsBucketResponseStream :: MonadAWS m
+  => ListObjectsV2
+  -> ConduitM a ListObjectsV2Response m ()
```

Comment:
Have you considered using a `TChan`?

Reply:
instead of Conduit?

Reply:
Yeah. We basically just need something that blocks until a thing comes in, and a conduit is more complex than we potentially need.